Automattic/pocket-casts-ios
# Migrate Crash reporting from Crashlytics to Sentry

Fixes #341 

Removes Crashlytics in favor of Sentry.

## To test
You'll need to update your APICredential file to proceed. 

1.) Switch your scheme to "Release"
2.) Add a `fatalError` exception to the app 
3.) Build and run.
4.) Stop the run.
5.) Start the app again by selecting the icon on the home screen.
6.) Execute the code that has the `fatalError` or perform a [crash](https://github.com/Automattic/pocket-casts-ios/issues?q=is%3Aopen+is%3Aissue+label%3Acrash) repro steps.
7.) Check Sentry for the crash and expect Crashlytics to not have the crash.

**Opt-out**
1.) Set privacy to Opt-out.
2.) Perform the steps above
3.) Expect Sentry to not contain the crash. 

## Checklist
